### PR TITLE
Fixes #41 - Add dependencies to gemspec

### LINKED ISSUE #31: specfile/%install should precompile assets

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by shlomizadok** [comment]
@domcleal -- would love if you could take a look and see if done right. Thanks

File: extra/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -109,6 +108,8 @@ popd
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_dir}/%{gem_name}.rb
+%{foreman_assets_plugin}
+%{gem_instdir}/public/assets/%{gem_name}
```
Comment (domcleal) [inline extra/rubygem-foreman_openscap.spec]: dupe of 111?

**Comment by domcleal** [comment]
Does it build?  What's the output from `rpm -qpl ...rpm`?

File: extra/rubygem-foreman_openscap.spec (hunk 1 above)
Comment (shlomizadok) [inline extra/rubygem-foreman_openscap.spec]: Actually cowardly copied from https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rubygem-foreman_docker/rubygem-foreman_docker.spec#L101-L102

Comment (domcleal) [inline extra/rubygem-foreman_openscap.spec]: Both look wrong then..

**Comment by shlomizadok** [comment]
@domcleal we're getting http://paste.fedoraproject.org/171843/14217513/ -- not sure what is wrong :/

**Comment by domcleal** [comment]
Even since ede8f37?

**Comment by isimluk** [comment]
On my rhel6 box, I am still getting
```
+ export GEM_PATH=/opt/rh/ruby193/root/usr/share/gems:/root/rpmbuild/BUILDROOT/ruby193-rubygem-foreman_openscap-0.2.0-1.el6.x86_64/opt/rh/ruby193/root/usr/share/gems
+ GEM_PATH=/opt/rh/ruby193/root/usr/share/gems:/root/rpmbuild/BUILDROOT/ruby193-rubygem-foreman_openscap-0.2.0-1.el6.x86_64/opt/rh/ruby193/root/usr/share/gems
+ cp /root/rpmbuild/BUILDROOT/ruby193-rubygem-foreman_openscap-0.2.0-1.el6.x86_64/usr/share/foreman/bundler.d/foreman_openscap.rb ./bundler.d/foreman_openscap.rb
+ unlink tmp
+ export BUNDLER_EXT_NOSTRICT=1
+ BUNDLER_EXT_NOSTRICT=1
+ /usr/bin/ruby193-rake 'plugin:assets:precompile[foreman_openscap]' RAILS_ENV=production --trace
** Invoke plugin:assets:precompile (first_time)
** Execute plugin:assets:precompile
rake aborted!
undefined method `[]' for nil:NilClass
/root/rpmbuild/BUILD/foreman_openscap-0.2.0/usr/share/foreman/lib/tasks/plugin_assets.rake:48:in `compile_assets'
/root/rpmbuild/BUILD/foreman_openscap-0.2.0/usr/share/foreman/lib/tasks/plugin_assets.rake:90:in `block in <top (required)>'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:205:in `call'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:205:in `block in execute'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:200:in `each'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:200:in `execute'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:158:in `block in invoke_with_call_chain'
/opt/rh/ruby193/root/usr/share/ruby/monitor.rb:211:in `mon_synchronize'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:151:in `invoke_with_call_chain'
/opt/rh/ruby193/root/usr/share/ruby/rake/task.rb:144:in `invoke'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:116:in `invoke_task'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:94:in `block (2 levels) in top_level'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:94:in `each'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:94:in `block in top_level'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:133:in `standard_exception_handling'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:88:in `top_level'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:66:in `block in run'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:133:in `standard_exception_handling'
/opt/rh/ruby193/root/usr/share/ruby/rake/application.rb:63:in `run'
/opt/rh/ruby193/root/usr/bin/rake:32:in `<main>'
Tasks: TOP => plugin:assets:precompile
```

**Comment by shlomizadok** [comment]
@ares, @isimluk  - May we remove the spec file from here, as @ares is doing that in foreman-packaging? I think we should only test & merge the changes at  foreman_openscap.gemspec and engine.rb. What do you think?

**Comment by isimluk** [comment]
Yep. We can scratch issue #31. And keep only gemspec commits here.

But please do not remove spec files yet. I am using them until the theforeman/foreman-packaging#499 is merged. We will remove the spec files from here afterwards.

**Comment by shlomizadok** [comment]
squashed & removed rubygem..spec changes as they are fixes in foreman-packaging.

**Comment by isimluk** [comment]
Thanks!

**Pull request merged**